covalent/_workflow/electron.py

```diff
@@ -30,7 +30,7 @@
 from .._file_transfer.enums import Order
 from .._file_transfer.file_transfer import FileTransfer
 from .._shared_files import logger
-from .._shared_files.context_managers import active_lattice_manager
+from .._shared_files.context_managers import active_dispatch_info_manager, active_lattice_manager
```

Comment:
If we're going to use `active_dispatch_info_manager` even for a short while, let's rename it to something more relevant to what it's doing.